src/utils/serde.rs

```diff
@@ -0,0 +1,3 @@
+pub(crate) fn _true() -> bool {
```

Comment:
Do we need this?

Reply:
It is used to configure some serialisation cases with `#[serde(default=_true")]` it requires function or Default trait.